addons/packages/harbor/2.5.3/bundle/config/overlay/add-rbac.yaml

```diff
@@ -0,0 +1,33 @@
+#@ load("@ytt:overlay", "overlay")
+#@ load("@ytt:template", "template")
+#@ load("/values.star", "values")
+#@ load("/globals.star", "setClusterRoleName")
+
+#@ if/end values.pspNames:
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: #@ setClusterRoleName()
+rules:
+- apiGroups:
+  - policy
+  resources:
+  - podsecuritypolicies
+  verbs:
+  - use
+
+#@ if/end values.pspNames:
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: #@ setClusterRoleName()
```

Comment:
`setClusterRoleName` => getClusterRoleBindingName

Reply:
We can add two functions in the globals.star.

One is getClusterRoleName, another is getClusterRoleBindingName

Reply:
I am using this function to concat both `ClusterRole`  name and `ClusterRoleBinding` name